Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -32,41 +31,27 @@
 /**
  * The default web router
  *
- * @Flow\Scope("singleton")
```

Comment:
Right, I don't like them :)
But that's not the reason for removing it here. By making the router depending on its constructor arguments it can no longer be a global singleton. This would lead to an exception during proxy building and it wouldn't make sense either.
Instead, the `RouterInterface` (i.e. its default implementation) now is the global singleton (and that can only be done via `Objects.yaml`)